src/bedtools/bedtools_bamtobed/script.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+## VIASH START
+## VIASH END
+
+# Unset parameters
+[[ "$par_bedpe" == "false" ]] && unset par_bedpe
+[[ "$par_mate1" == "false" ]] && unset par_mate1
+[[ "$par_bed12" == "false" ]] && unset par_bed12
+[[ "$par_split" == "false" ]] && unset par_split
+[[ "$par_splitD" == "false" ]] && unset par_splitD
+[[ "$par_edit_distance" == "false" ]] && unset par_edit_distance
+[[ "$par_tag" == "false" ]] && unset par_tag
+[[ "$par_color" == "false" ]] && unset par_color
+[[ "$par_cigar" == "false" ]] && unset par_cigar
```

Comment:
```suggestion
unset_if_false=( 
  par_bedpe
  par_mate1
  par_bed12
  par_split
  par_splitD
  par_edit_distance
  par_tag
  par_color
  par_cigar
)

for par in ${unset_if_false[@]}; do
    test_val="${!par}"
    [[ "$test_val" == "false" ]] && unset $par
done
```

Reply:
Could I keep it as it is?

Reply:
Or how about this?

```suggestion
unset_if_false=( 
  par_bedpe
  par_mate1
  par_bed12
  par_split
  par_splitD
  par_edit_distance
  par_tag
  par_color
  par_cigar
)

for par in ${unset_if_false[@]}; do
    [[ "${!par}" == "false" ]] && unset $par
done
```

Reply:
@tgaspe What do you not like about @DriesSchaumont 's suggestion? :)

Reply:
It was just a stylistic choice of my part.